com.unity.render-pipelines.core/Editor/LookDev/DisplayWindow.cs

```diff
@@ -625,6 +625,19 @@ IStyle GetEnvironmentContenairDraggerStyle()
             }
         }
 
+        void Update()
+        {
+            // [case 1245086] Guard in case the SRP asset is set to null (or to a not supported SRP) when the lookdev window is already open
+            // Note: After an editor reload, we might get a null OnUpdateRequestedInternal and null SRP for a couple of frames, hence the check.
+            if (!LookDev.supported && OnUpdateRequestedInternal != null)
+            {
+                // Print an error and close the Lookdev window (to avoid spamming the console)
+                Debug.LogError($"LookDev is not supported by this Scriptable Render Pipeline: "
+                    + (RenderPipelineManager.currentPipeline == null ? "No SRP in use" : RenderPipelineManager.currentPipeline.ToString()));
+                LookDev.Close();
+            }
+        }
+
```

Comment:
Isn't this change part of another PR ?

Reply:
Yep, it's mentioned in the PR description :) We need to merge them in order.